doc/modules/model_evaluation.rst

```diff
@@ -707,17 +707,21 @@ The :func:`hamming_loss` computes the average Hamming loss or `Hamming
 distance <https://en.wikipedia.org/wiki/Hamming_distance>`_ between two sets
 of samples.
 
-If :math:`\hat{y}_j` is the predicted value for the :math:`j`-th label of
-a given sample, :math:`y_j` is the corresponding true value, and
-:math:`n_\text{labels}` is the number of classes or labels, then the
-Hamming loss :math:`L_{Hamming}` between two samples is defined as:
+If :math:`\hat{y}_{i,j}` is the predicted value for the :math:`j`-th label of a
+given sample point :math:`i`, :math:`y_{i,j}` is the corresponding true value,
+:math:`n_\text{samples}` is the number of sample points and
+:math:`n_\text{labels}` is the number of labels, then the Hamming loss 
+:math:`L_{Hamming}` is defined as:
 
 .. math::
 
-   L_{Hamming}(y, \hat{y}) = \frac{1}{n_\text{labels}} \sum_{j=0}^{n_\text{labels} - 1} 1(\hat{y}_j \not= y_j)
+   L_{Hamming}(y, \hat{y}) = \frac{1}{n_\text{samples} * n_\text{labels}} \sum_{i=0}^{n_\text{samples}-1} \sum_{j=0}^{n_\text{labels} - 1} 1(\hat{y}_{i,j} \not= y_{i,j})
 
 where :math:`1(x)` is the `indicator function
-<https://en.wikipedia.org/wiki/Indicator_function>`_. ::
+<https://en.wikipedia.org/wiki/Indicator_function>`_.
+
+In multiclass classification, the Hamming loss is computed as the average of the
+0-1 classification loss (:math:`L_{0-1}`) over :math:`n_{\text{samples}}`. ::
```

Comment:
I noticed that the multiclass case is better explained in a note a bit below. Maybe we can revert this change and just mention that the equation above does not hold for the multiclass case, then refer to (or even move) the note below.

Reply:
I have added a line under the equation mentioning the limitation and also pointed the user to the note for more information.